.github/workflows/deploy.yml

```
@@ -0,0 +1,122 @@
+      run: |
+        cd backend
+        uv sync
+    
+    - name: Run tests
+      run: |
+        cd backend
+        uv run pytest
+    
+    - name: Setup Google Cloud SDK
+      if: github.ref == 'refs/heads/main'
+      uses: google-github-actions/setup-gcloud@v1
```

actionlint (1.7.7)

the runner of "google-github-actions/setup-gcloud@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)